serveradmin/common/templates/opensearch.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
+<ShortName>ServerAdmin</ShortName>
+<Description>Server shell</Description>
```

Comment:
Servershell is also written together.  `Description` should probably say something else anyway.